app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,72 @@
+  <body class="signup-body">
+    <div class="container py-5">
+      <div class="row justify-content-center">
+        <div class="col-md-6">
+          <div class="card shadow">
+            <div class="card-body">
+            <header class="header-class">
+                <%= link_to root_path, data: { turbo: false }  do %>
+                  <%= inline_svg_tag "chevron_left.svg", class: "chevron" %>
+                <% end %>                  
+              <div class="header-title-wrapper">
+                <h2 class="mb-4 header-title"><i class="fas fa-user-plus"></i> Sign up</h2>
+              </div>
+              </header>
+
+              <turbo-frame id="signup_form">
+                <%= form_with(model: resource, as: resource_name, url: registration_path(resource_name), html: { class: 'mt-4' }) do |f| %>
+                  <%= render "devise/shared/error_messages", resource: resource %>
+
+                  <div class="mb-3">
+                    <%= f.label :name, class: 'form-label' %>
+                    <div class="input-group">
+                      <span class="input-group-text"><i class="fas fa-user"></i></span>
+                      <%= f.text_field :name, autofocus: true, autocomplete: "name", class: 'form-control', placeholder: 'Enter your name' %>
+                    </div>
+                  </div>
+
+                  <div class="mb-3">
+                    <%= f.label :email, class: 'form-label' %>
+                    <div class="input-group">
+                      <span class="input-group-text"><i class="fas fa-envelope"></i></span>
+                      <%= f.email_field :email, autocomplete: "email", class: 'form-control', placeholder: 'Enter your email' %>
+                    </div>
+                  </div>
+
+                  <div class="mb-3">
+                    <%= f.label :password, class: 'form-label' %>
+                    <div class="input-group">
+                      <span class="input-group-text"><i class="fas fa-lock"></i></span>
+                      <%= f.password_field :password, autocomplete: "new-password", class: 'form-control', placeholder: 'Enter your password' %>
+                    </div>
+                  </div>
+
+                  <div class="mb-3">
+                    <%= f.label :password_confirmation, class: 'form-label' %>
+                    <div class="input-group">
+                      <span class="input-group-text"><i class="fas fa-lock"></i></span>
+                      <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'form-control', placeholder: 'Confirm your password' %>
+                    </div>
+                  </div>
+
+                  <div class="form-actions text-center">
+                    <%= f.submit "Sign up", class: 'signup-button btn-lg px-5' %>
+                  </div>
+                <% end %>
+              </turbo-frame>
+
+              <%= render "devise/shared/links" %>
+            </div>
```

Comment:
- [ ] Great job working on the splash screen but I think it would be great if you fix the inputs in the login as mentioned by the previous reviewer since your design should be the same as the given design so kindly work on it to make it the same the inputs should be close to each other and the next button should be in the navbar 🔥💪🏻♻

- Your design
![image](https://github.com/yusufyusie/budget_application/assets/58553711/ce04ca63-1207-4df5-b600-fdb18446a558)

- Given
![image](https://github.com/yusufyusie/budget_application/assets/58553711/f091c585-3b70-4f71-abf3-88a4adb74095)